### DISCUSSION TITLE
How to add i18n support?

### DISCUSSION THREAD

**Discussion by tarcisiodelmondes** [hugo-sid/hugo-blog-awesome#63] [Q&A]
I've been trying to add multilingual support to my blog, but to no avail. It should probably be something simple, but since I'm new to Hugo I've been having difficulty with this part.

**Comment by hugo-sid** [comment]
https://gohugo.io/content-management/multilingual/

**Reply by tarcisiodelmondes** [reply]
Had already looked at the documentation, but I found an error here. When I created the blog using `hugo new site myblog` Hugo created a 'hhugo.toml' file that conflicted with my 'config.toml.' Deleting the 'hugo.toml' file, which was automatically created by Hugo, solved my problem. Now the i18n is working correctly. But thanks anyway.
